tests/unit/integrations/neural_prophet/materializers/test_neural_prophet_materializer.py

```diff
@@ -0,0 +1,59 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from contextlib import ExitStack as does_not_raise
+
+import numpy as np
+import pandas as pd
+from neuralprophet import NeuralProphet
+
+from zenml.integrations.neural_prophet.materializers.neural_prophet_materializer import (
+    NeuralProphetMaterializer,
+)
+from zenml.pipelines import pipeline
+from zenml.steps import step
+
+
+def test_neural_prophet_booster_materializer(clean_repo):
+    """Tests whether the steps work for the Neural Prophet forecaster materializer."""
+
+    @step
+    def read_forecaster() -> NeuralProphet:
+        """Reads and materializes a Neural Prophet forecaster."""
+        sample_df = pd.DataFrame(
+            {
+                "ds": pd.date_range("2018-01-01", periods=10),
+                "y": np.random.rand(10),
+            }
+        )
+        m = NeuralProphet(epochs=7, batch_size=37)
+        m.fit(sample_df)
```

Comment:
how long does this run?

Reply:
if it's not instant I'd reduce epochs to either 1 or 0

Reply:
I'll reduce it, though it takes basically no time at all to run. It can't be 0, I think, based on the errors I was getting last week. I've reduced it to 2 now.